src/Pages/Home.jsx

```diff
@@ -122,173 +123,52 @@ class Home extends React.Component {
-                    />
-                  </a>
-                  <a
-                    target='_blank'
-                    rel='noreferrer'
-                    href='https://www.linkedin.com/in/kishor-ghising-9b77611a4/'
-                  >
-                    <Icon
-                      className='social-media-icon'
-                      name={['fab', 'linkedin']}
-                      size='2x'
-                    />
-                  </a>
-                  <a
-                    target='_blank'
-                    rel='noreferrer'
-                    href='https://www.facebook.com/kghisinga/'
-                  >
-                    <Icon
-                      className='social-media-icon'
-                      name={['fab', 'facebook']}
-                      size='2x'
-                    />
-                  </a>
-                </div>
-              </div>
-              <div className='team_member mb-3'>
-                <div className='member_img'>
-                  <img src='/public/images/shrawan.jpg' />
-                </div>
-                <h3>Shrawan Ghimire</h3>
-                <div className='social_media mt-2'>
-                  <a
-                    target='_blank'
-                    rel='noreferrer'
-                    href='https://www.instagram.com/shrwan_ghimire/'
-                  >
-                    <Icon
-                      className='social-media-icon'
-                      name={['fab', 'instagram']}
-                      size='2x'
-                    />
-                  </a>
-                  <a
-                    target='_blank'
-                    rel='noreferrer'
-                    href='https://www.linkedin.com/in/shrawan-ghimire-36536b145/'
-                  >
-                    <Icon
-                      className='social-media-icon'
-                      name={['fab', 'linkedin']}
-                      size='2x'
-                    />
-                  </a>
-                  <a
-                    target='_blank'
-                    rel='noreferrer'
-                    href='https://www.facebook.com/shrawan.ghimire.7/'
-                  >
-                    <Icon
-                      className='social-media-icon'
-                      name={['fab', 'facebook']}
-                      size='2x'
-                    />
-                  </a>
-                </div>
-              </div>
+          <div className='our-team-section'>
+            <h1>Our Team</h1>
+            <div className='team-member'>
+              <TeamMemberCard
+                imgLink='/public/images/anish.jpg'
+                name='Anish Silwal Khatri'
+                description='Anish Sliwal Khatri is a Software Engineer at Leapfrog Technology. He is pursuing 
+                  his bachelors degree in Bsc.CSIT at Bhaktapur Multiple Campus. He has a huge
+                  interest in Game Development. He is a massive football fan (supports Chelsea Football Club).'
```

Comment:
The text is to big. Makes the icon overflow.
```suggestion
                description='Anish Sliwal Khatri is a Software Engineer at Leapfrog Technology. He is pursuing 
                  his bachelors degree in Bsc.CSIT. He has a huge
                  interest in Game Development. He is a massive football fan (supports Chelsea Football Club).'
```